dev-env/windows/libexec/core.ps1

```diff
@@ -28,6 +28,8 @@ function da_reset_path {
 
 function da_enable_win_dev_env {
     da_enable_scoop
+    # also add dadew to the PATH
+    $env:PATH = "$PSScriptRoot\..\bin;$env:PATH"
```

Comment:
I combined it into: https://github.com/digital-asset/daml/blob/master/dev-env/windows/libexec/core.ps1#L20